package.json

```diff
@@ -64,6 +64,7 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.4.0",
     "morgan": "1.2.0",
+    "ms": "0.7.1",
```

Comment:
I do like the ms microlib, but I think you'll want to get another +1 before adding this. Other folks would like us to create a time constant module.